content/docs/trust/trust-manager/README.md

```diff
@@ -88,18 +91,25 @@ spec:
 - `inLine` - a manually specified string containing at least one certificate
 - `useDefaultCAs` - usually, a bundle of publicly trusted certificates
 
-These sources, along with the single currently supported target type (`configMap`)
-are documented in the trust-manager [API reference documentation](./api-reference.md).
+`ConfigMap` is the default target type, but as of v0.7.0 trust-manager also supports `Secret` resources as targets.
+
+Support for `Secret` targets must be explicitly enabled in the trust-manager controller; see details below under "Enable Secret targets".
+
+All sources and target options are documented in the trust-manager [API reference documentation](./api-reference.md).
 
 #### Targets
 
-All `Bundle` targets are written to `ConfigMap`s whose name matches that of the `Bundle`, and every
-target has a PEM-formatted bundle included.
+All `Bundle` targets are written to `ConfigMap`s (and/or `Secret`s) whose name matches that of the
+`Bundle`, and every target has a PEM-formatted bundle included.
+
+Users can also optionally choose to write JKS/PKCS#12 formatted binary trust store(s) to targets.
+JKS has been supported since v0.5.0, and PKCS#12 since v0.7.0.
```

Comment:
> [spelling]     content/docs/trust/trust-manager/README.md
[spelling]       106 | nce v0.5.0, and PKCS#12 since v0.7.0.

Why? How to fix this? Add "v0.7.0." to the dictionary?

Reply:
That'll probably be it yeah 😦 `./scripts/verify` should show locally with highlighting what specifically is broken